packages/flutter/lib/src/rendering/semantics_event.dart

```diff
@@ -25,15 +25,27 @@ abstract class SemanticsEvent {
 
   /// Converts this event to a Map that can be encoded with
   /// [StandardMessageCodec].
-  Map<String, dynamic> toMap();
+  Map<String, dynamic> toMap({int nodeId}) {
+    final Map<String, dynamic> event = <String, dynamic>{
+      'type': type,
+      'data': getDataMap(),
+    };
+    if (nodeId != null) {
+      event['nodeId'] = nodeId;
+    }
+    return event;
+  }
+
+  // Returns the event's data object.
```

Comment:
Three leading ///.

Should also mention that the returned obj needs to be encodable with [StandardMessageCodec].